tutorials/24_Building_Chat_App.ipynb

```diff
@@ -117,7 +117,7 @@
       "source": [
```

Comment:
<blockquote>offers <a href="https://huggingface.co/docs/api-inference/index" target="_blank">hosted Inference API</a> which</blockquote><p>offers a hosted Inference API, which</p><blockquote>To use service</blockquote><p>To use the service</p><p> </p><p>Is there a line break after "API key"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
No, there's no line break after API Key. Should there be? 

Fixed the other comments btw